index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Bungee+Spice&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:wght@300;400;700;900&family=Lobster+Two:wght@400;700&family=Poppins:wght@300;400;500;600;700;800&family=Satisfy&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />  <link rel="stylesheet" href="style.css">
+  <title>Web development course</title>
+</head>
+<body>
+  <div class="head-bar">
+    <div class="head-bar-content">
+      <img src="assets/images//facebook.svg" alt="" class="head-bar-margin">
+       <p><a href="" class="links">English</a></p>
+       <p><a href="" class="links">Sign up</a></p>
+       <p><a href="" class="links">Log in</a></p>
+     </div>
+  </div>
+   <div class="wrapper">
+
+    <div class="navigation">
+      <ul class="nav-tag">
+        <li><a href="index.html" class="logo">House Of Technology</a></li>
+        <li><a href="about.html" class="active">About</a></li>
+        <li><a href="">Program</a></li>
+        <li><a href="">Join</a></li>
+        <li><a href="">Sponsor</a></li>
+        <li><a href="">News</a></li>
+        <li><button class="btn-1">Tech Campaign</button></li>
+      </ul>
+    </div>
+
+    <div id="navbar">
+      <ul class="nav-menu">
+          <li class="nav-item nav-area"><a href="about.html" class="nav-link">About</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+      </ul>
+
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+  </div>
+      
+  
+    <div class="bg-image">
+      <div class="main-section">
+        <div class="heading">
+          <p class="header-paragraph">"Computer Workshop"</p>
+          <h1>MAINTENANCE AND SOFTWARE INSTALLATION</h1>
+        </div>
```

Comment:
- [OPTIONAL] Kindly make sure you add a background image for this title and use a bigger font weight to make it as per the design to meet the project requirements :recycle: .
- Your design:
![BbgiMG](https://user-images.githubusercontent.com/97834160/210795066-f5d54da1-6da0-4a9a-80b6-a08b8c75e052.png)
- Behance design:
![bgImg](https://user-images.githubusercontent.com/97834160/210795173-488ec4d2-dd9b-489e-a019-ad0165594a9e.png)